samples/matter/contact_sensor/sysbuild/mcuboot/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# TODO: Workaround, disable memory guard to avoid false faults in application after boot
+CONFIG_HW_STACK_PROTECTION=n
+
+CONFIG_BOOT_WATCHDOG_FEED=n
+
+# nRF54L15DK uses SPI NOR external flash
+CONFIG_GPIO=y
+CONFIG_SPI=y
+CONFIG_SPI_NOR=y
+CONFIG_SPI_NOR_SFDP_DEVICETREE=y
+CONFIG_SPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
+
+CONFIG_PM_OVERRIDE_EXTERNAL_DRIVER_CHECK=y
```

Comment:
```suggestion
```
this should be set at sysbuild level which then sets it in mcuboot, putting this here does nothing